apps/etterlatte-saksbehandling-ui/client/src/shared/api/vilkaarsvurdering.ts

```diff
@@ -138,8 +141,10 @@ export interface VilkaarsvurderingVurdertResultat {
 export interface MigrertYrkesskadefordel {
   migrertYrkesskadefordel: boolean
 }
-
-export enum VilkaarType {
-  BP_FORMAAL_2024 = 'BP_FORMAAL_2024',
-  BP_DOEDSFALL_FORELDER_2024 = 'BP_DOEDSFALL_FORELDER_2024',
+export interface Vilkaartyper {
+  typer: Vilkaartype[]
+}
+export interface Vilkaartype {
+  name: string
+  tittel: string
```

Comment:
Hvorfor kan ikke dette være en Enum? Hvis ikke så er vel `Vilkaartyper` forsatt overflødig, der man bare kan bruke typen `Array<Vilkaartype>` hvor det trengs

Reply:
Det kan vera ein enum, og det er det backend, men den er svær der (`VilkaarType.kt`), så eg hadde ikkje så lyst til å redeklarere heile den i frontend.

Reply:
Av ein eller annan grunn gir `Array<Vilkaartype>` meg `typer.map is not a function`, utan at eg heilt skjønner korfor.

Reply:
Aha, ja det gir det sikkert mening hvis det blir en del vilkaarstyper